Makefile

```diff
@@ -180,10 +180,7 @@ run-apk-tests:
 	exit $$_r
 
 run-performance-tests:
-	_r=0 ; \
-	$(call MSBUILD_BINLOG,run-apk-tests,,Test) $(TEST_TARGETS) /t:RunApkTests /p:RunApkTestsTarget=RunPerformanceApkTests $(APK_TESTS_PROP) || _r=1 ; \
```

Comment:
Not sure why these changes are needed as part of a .NET Core installation change.

Reply:
I was mainly trying to get the build log shorter 👀 

It seems both the startup & MSBuild perf tests fail to build, which is odd.